test/wg-easy/1.0.0/questions.yaml

```diff
@@ -0,0 +1,259 @@
+groups:
+  - name: Configuration
+    description: WG-Easy application configuration
+  - name: Storage
+    description: Configure storage for WG-Easy
+  - name: Networking
+    description: Networking Configuration for WG-Easy
+  - name: Advanced DNS Settings
+    description: Configure DNS settings
+  - name: Resource Limits
+    description: Set CPU/memory limits for Kubernetes Pod
+
+portals:
+  web_portal:
+    protocols:
+      - http
+    host:
+      - $node_ip
+    ports:
+      - $variable-web_port
```

Comment:
Admin portal will be broken i assume as `web_port` is not a valid variable in schema

Reply:
Fixed!